airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/defaults/custom_reports_schema.json

```diff
@@ -21,6 +21,12 @@
           "type": "string"
         }
       },
+      "offset": {
+        "type": ["integer", "string"]
```

Comment:
What is the data type inside of that field? We should not have both types and expect both, please use the only type that fits the data as follows:
```json
"type": ["null", "integer"]
```